datasourcevariable/schemas/migrate/migrate.cue

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migrate
+
+#grafanaType: "text"
```

Comment:
This line should be removed. `#grafanaType` works only in the case of panel plugins (https://github.com/perses/perses/blob/main/docs/plugins/cue.md)